core/sqf/monitor/linux/pnode.cxx

```diff
@@ -1528,6 +1528,14 @@ void CNodeContainer::AddNodes( )
     int pnid;
     int rank;
     int *sparePNids = NULL;
+    
+    //  only relevant on a workstation acting as a single node
+    const char* envVar = getenv("SQ_MAX_RANK"); 
+    int maxNode;
+    if (envVar != NULL)
+      maxNode = atoi (envVar);
```

Comment:
Again, I suppose SQ_MAX_RANK is a debugging tool, so we don't bother checking for non-numeric inputs such as might happen with "export SQ_MAX_RANK=ON" (where atoi would return 0 I think).